src/dotnes.tasks/Utilities/Transpiler.cs

```diff
@@ -387,7 +388,9 @@ void GetUsedMethods(MetadataReader reader)
         }
 
         if (neslib is null)
-            throw new InvalidOperationException("Did not find TypeReference to NES.NESLib!");
+            // We don't have NES.NESLib, but that's fine
+            // since we can be generic NES transpiler and that's useful for testing transpiler itself.
+            return;
```

Comment:
Can you share a quick example of what you're working around here? Example of an error message?

Does `FSharp.Core.dll` show up here in F# projects?

Reply:
I did try to create smallest possible app without peripherals for testing compiler. You can nuke it if not like, or we can discuss it in more details on some other issue